core_perception/lidar_localizer/nodes/ndt_matching/ndt_matching.cpp

```diff
@@ -224,6 +211,7 @@ static bool _use_imu = false;
 static bool _use_odom = false;
 static bool _imu_upside_down = false;
 static bool _output_log_data = false;
+static std::string _output_tf_frame_id = "base_link";
```

Comment:
This file was previously modified to allow the map and base_link to be specified as parameters.
See this commit
https://github.com/usdot-fhwa-stol/autoware.ai/commit/c55ee500ca23e8248bcbf3a7f5f8fcbf38b2b26c#diff-1a554d23d2cf99ef8442ec85bc62138dff6a271a6875155071b8000a4d05614e

Please update this change set to keep this possible.

Reply:
Just made the map configurable, I think the base frame is already a parameter.